src/pages/Profile.tsx

```diff
@@ -0,0 +1,145 @@
+import "../index.css";
+import { useForm } from "@mantine/form";
+import {
+  PasswordInput,
+  Text,
+  Paper,
+  Group,
+  PaperProps,
+  Button,
+} from "@mantine/core";
+import axios from "axios";
+import { IconCheck, IconX } from "@tabler/icons-react";
+import { notifications } from "@mantine/notifications";
+
+export default function Profile(props: PaperProps) {
+  const form = useForm({
+    initialValues: {
+      currentPassword: "",
+      newPassword: "",
+      confirmNewPassword: "",
+    },
+
+    validate: {
+      currentPassword: (val) =>
+        val.length <= 1 ? "Current Password should not be empty" : null,
+      newPassword: (val) =>
+        val.length <= 4
+          ? "Password should include at least 4 characters"
+          : null,
+      confirmNewPassword: (val, values) =>
+        val === values.newPassword
+          ? null
+          : "Password confirmation should match the the new password",
+    },
+  });
+
+  return (
+    <div className="profile-container">
+      <div>
+        <Text>Ian Kamau</Text>
+        <Text>kian99564@gmail.com</Text>
+        <Text>Ian Kamau</Text>
+      </div>
+
+      <Paper
+        radius="md"
+        p="xl"
+        {...props}
+        style={{
+          width: "30%",
+        }}
+      >
+        <Text size="lg" weight={500}>
+          Your Profile Page
+        </Text>
+        <form
+          onSubmit={form.onSubmit(() => {
+            axios({
+              method: "post",
+              url: "http://localhost:3000/api/v1/auth/update/password",
```

Comment:
Remove this from here. Instead, use .env files to store such variables so that it is easy to modify the endpoint. You can have something like this in your .env file

```bash
# Declare the server URL here as below
SERVER_URL=http://localhost:3000/api/v1
```

Then in your code, instead of having the endpoint hardcoded, you can simply use

```javascript
{
      // ... other parts of your code

      url: ${process.env.SERVER_URL}/auth/update/password

     //... more code
}
```